Converters/Tekla/Speckle.Converter.Tekla2024/ToSpeckle/Helpers/ReportHandler.cs

```diff
@@ -0,0 +1,88 @@
+﻿namespace Speckle.Converter.Tekla2024.ToSpeckle.Helpers;
+
+public class ReportPropertyHandler
+{
+  private static readonly Dictionary<Type, string[]> s_typeSpecificProperties =
+    new()
+    {
+      {
+        typeof(TSM.Beam),
+        new[]
+        {
+          "VOLUME",
+          "WEIGHT",
+          "LENGTH",
+          "HEIGHT",
+          "WIDTH",
+          "AREA",
+          "PROFILE_TYPE",
+          "MATERIAL_TYPE",
+          "CLASS",
+          "PREFIX",
+          "ASSEMBLY_POS",
+          "ASSEMBLY_NAME",
+          "PHASE"
+        }
+      },
+      {
+        typeof(TSM.ContourPlate),
+        new[]
+        {
+          "VOLUME",
+          "WEIGHT",
+          "AREA",
+          "PROFILE_TYPE",
+          "MATERIAL_TYPE",
+          "CLASS",
+          "PREFIX",
+          "ASSEMBLY_POS",
+          "PHASE"
+        }
+      },
+      { typeof(TSM.RebarGroup), new[] { "NUMBER_OF_REBARS", "TOTAL_LENGTH", "WEIGHT", "SIZE", "GRADE", "CLASS" } },
+      { typeof(TSM.SingleRebar), new[] { "LENGTH", "WEIGHT", "SIZE", "GRADE", "CLASS" } },
+      { typeof(TSM.BoltArray), new[] { "BOLT_SIZE", "NUMBER_OF_BOLTS", "BOLT_STANDARD", "BOLT_TYPE", "LENGTH" } }
+    };
+
+  public Dictionary<string, object?> GetProperties(TSM.ModelObject modelObject)
```

Comment:
if it is in converter, I would rather to follow "converter" logic instead random class and function names if we don't have a strong reason. lacking context shout me if im wrong